sygnal/firebasepushkin.py

```diff
@@ -13,27 +13,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import json
 import logging
-from typing import Dict, Optional
+from typing import Dict
+from functools import partial
 
 import attr
-from firebase_admin import credentials, initialize_app, messaging
+from opentracing import logs
+from firebase_admin import (
```

Comment:
I've just noticed neither `attr` nor `firebase_admin` are in `setup.py`. Could you add the appropriate versions in there please?